src/h5web/providers/hsds/utils.test.ts

```diff
@@ -10,220 +10,222 @@ import type {
 } from './models';
 import { convertHsdsType, parseComplex } from './utils';
 
-describe('HsdsProvider utilities', () => {
-  describe('convertHsdsType', () => {
-    interface TestType {
-      hsds: HsdsType;
-      hdf5: HDF5Type;
-    }
-
-    const asciiStrType: TestType = {
-      hsds: {
-        class: HDF5TypeClass.String,
-        charSet: 'H5T_CSET_ASCII',
-        strPad: 'H5T_STR_NULLPAD',
-        length: 25,
-      },
-      hdf5: {
-        class: HDF5TypeClass.String,
-        charSet: 'ASCII',
-        length: 25,
-      },
-    };
-    const unicodeStrType: TestType = {
-      hsds: {
-        class: HDF5TypeClass.String,
-        charSet: 'H5T_CSET_UTF8',
-        strPad: 'H5T_STR_NULLTERM',
-        length: 49,
-      },
-      hdf5: {
-        class: HDF5TypeClass.String,
-        charSet: 'UTF8',
-        length: 49,
-      },
-    };
+interface TestType {
+  hsds: HsdsType;
+  hdf5: HDF5Type;
+}
+
+const asciiStrType: TestType = {
+  hsds: {
+    class: HDF5TypeClass.String,
+    charSet: 'H5T_CSET_ASCII',
+    strPad: 'H5T_STR_NULLPAD',
+    length: 25,
+  },
+  hdf5: {
+    class: HDF5TypeClass.String,
+    charSet: 'ASCII',
+    length: 25,
+  },
+};
+
+const unicodeStrType: TestType = {
+  hsds: {
+    class: HDF5TypeClass.String,
+    charSet: 'H5T_CSET_UTF8',
+    strPad: 'H5T_STR_NULLTERM',
+    length: 49,
+  },
+  hdf5: {
+    class: HDF5TypeClass.String,
+    charSet: 'UTF8',
+    length: 49,
+  },
+};
```

Comment:
Following the same principles, `asciiStrType` and `unicodeStrType` could be inlined in the according tests.